lib/recurly/response.php

```diff
@@ -57,7 +57,7 @@ public function assertValidResponse()
         // Handled in assertSuccessResponse()
         return;
       case 429:
-        throw new Recurly_ApiRateLimitError('You have made too many API requests in the last hour. Future GET API requests will be ignored until the beginning of the next hour.');
+        throw new Recurly_ApiRateLimitError('You have made too many API requests in the last 5 minutes. Future API requests may be ignored until your rate limit resets in 5 minutes.');
```

Comment:
I think it's important to leave the `GET` in there, since my understanding is those are the only requests we limit.